### PR TITLE
chore: Implement healthcheck endpoint

### PR DESCRIPTION
**Pull request by rhysbarrett**
### JIRA

[Jira Ticket](https://companieshouse.atlassian.net/browse/SR-92)

### Change Description

Implements and tests a very basic healthcheck endpoint (`/suppress-my-details/healthcheck`, which returns status 200 (`OK`).

### Work Checklist

- [x] Tests added where applicable
- [x] Test coverage of new code meets target of 80%

### CODE REVIEW

File: src/controllers/HealthcheckController.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import { Response } from 'express';
+import { StatusCodes } from 'http-status-codes';
+
+export class HealthcheckController {
+
+  public healthcheck = (_: any, res: Response, __: any) => {
```
Comment (akaJSaunders) [inline src/controllers/HealthcheckController.ts:6]: what with the two unused parameters? Is that where request and next function would be? I think its best we follow the convention we set on other controllers

(req: Request, res: Response, next: NextFunction)

Comment (rhysbarrett) [reply src/controllers/HealthcheckController.ts:6]: Yeahhhh I don't like having named unused parameters but I suppose readability takes precedence. I'll put the proper names/types back.

**Review by akaJSaunders — APPROVED** [review]
Thanks! Good job! 🚀

**Review by akaJSaunders — APPROVED** [review]
Its so good im approving it twice

**Pull request merged**